### PR TITLE
test(e2e): created Jest API-tests for retrieving gateway information

### PR DESCRIPTION
**Pull request by ctschacher**
Jest API-tests for retrieving gateway information using the following endpoints:

/instances
/instances/:instanceId

gravitee-io/issues#7474

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/test-create-gateway-information-tests-run-e2e/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-e2e/api-test/apis/gateway-instances.spec.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import { GatewayApi } from '@management-apis/GatewayApi';
+import { forManagementAsAdminUser, forManagementAsApiUser } from '@client-conf/*';
+import { fail } from '../../lib/jest-utils';
+
+const orgId = 'DEFAULT';
+const envId = 'DEFAULT';
+const gateway_admin = new GatewayApi(forManagementAsAdminUser());
+const gateway_user = new GatewayApi(forManagementAsApiUser());
+
+describe('Gateway tests', () => {
```
Comment (gaetanmaisse) [inline gravitee-apim-e2e/api-test/apis/gateway-instances.spec.ts]: `Gateway tests` is a bit misleading as we are not testing the Gateway directly but the /gateway endpoint of the management API
```suggestion
describe('GatewayApi', () => {
```

File: gravitee-apim-e2e/api-test/apis/gateway-instances.spec.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+import { beforeAll, describe, expect, test } from '@jest/globals';
+import { GatewayApi } from '@management-apis/GatewayApi';
+import { forManagementAsAdminUser, forManagementAsApiUser } from '@client-conf/*';
+import { fail } from '../../lib/jest-utils';
+
+const orgId = 'DEFAULT';
+const envId = 'DEFAULT';
+const gateway_admin = new GatewayApi(forManagementAsAdminUser());
+const gateway_user = new GatewayApi(forManagementAsApiUser());
```
Comment (gaetanmaisse) [inline gravitee-apim-e2e/api-test/apis/gateway-instances.spec.ts]:
```suggestion
const gatewayApiAsAdmin = new GatewayApi(forManagementAsAdminUser());
const gatewayApiAsUser = new GatewayApi(forManagementAsApiUser());
```

**Review by gaetanmaisse — COMMENTED** [review]
LGTM, I just did 2 comments related to naming

File: gravitee-apim-e2e/api-test/apis/gateway-instances.spec.ts (hunk 2 above)
Comment (ctschacher) [reply gravitee-apim-e2e/api-test/apis/gateway-instances.spec.ts]: I agree, that looks better.

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**